src/Symfony/Bundle/SecurityBundle/Resources/views/Collector/security.html.twig

```diff
@@ -2,39 +2,47 @@
 
 {% block toolbar %}
     {% if collector.tokenClass %}
-        {% set color_code = (collector.enabled and collector.authenticated) ? 'green' : 'yellow'  %}
-        {% set authentication_color_code = (collector.enabled and collector.authenticated) ? 'green' : 'red'  %}
-        {% set authentication_color_text = (collector.enabled and collector.authenticated) ? 'Yes' : 'No'  %}
+        {% set is_authenticated = collector.enabled and collector.authenticated  %}
+        {% set color_code = is_authenticated ? '' : 'yellow' %}
     {% else %}
-        {% set color_code = collector.enabled ? 'red' : 'black'  %}
+        {% set color_code = collector.enabled ? 'red' : '' %}
     {% endif %}
+
+    {% set icon %}
+        <svg version="1.1" xmlns="http://www.w3.org/2000/svg" x="0px" y="0px" height="24" viewBox="0 0 24 24" enable-background="new 0 0 24 24" xml:space="preserve">
+            <path fill="#AAAAAA" d="M21,20.4V22H3v-1.6c0-3.7,2.4-6.9,5.8-8c-1.7-1.1-2.9-3-2.9-5.2c0-3.4,2.7-6.1,6.1-6.1s6.1,2.7,6.1,6.1c0,2.2-1.2,4.1-2.9,5.2C18.6,13.5,21,16.7,21,20.4z"/>
+        </svg>
```

Comment:
I'd prefer to not do that. In the other icons make sense because we have a folder with lots of icons. In this case it woul be only 1 icon in 1 folder. I don't like that.